app/views/courses/confirmation.html.erb

```diff
@@ -79,6 +80,7 @@
                 ) %>
               </dd>
             </div>
```

Comment:
I recommend indenting the `div` tag inside the `unless` block

Reply:
Cheers I've fixed